packages/main/src/types/HasPopup.js

```diff
@@ -0,0 +1,62 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * Different types of HasPopup.
```

Comment:
Please verify:
* Different types of <code>HasPopup</code>.
or
* Different types of popups.